app/src/main/res/values/strings.xml

```diff
@@ -117,36 +117,33 @@
         like better. With enough comparison picks, the app will create a ranking from whom you like
         the most to least.\n
         \n
-        Next the app limits every user to maximum 10 matches. Matches are made based on whom you
+        Next the app limits every user to maximum 10 matches. You get matched based on whom you
         like the most and who likes you back. If you both are super popular, chances are you\'ll get
-        matched up unless either of you runs out of matches. If you are more average attractive,
-        chances are that your top choices will run out of matches before you\'d even get considered.
-        You\'ll likely end up matching with other average attractive people who couldn\'t get their
-        first choices.\n
+        matched up unless either of you runs out of matches.\n
         \n
-        There is of course the chance that you and your hottie are a niche. Say you are both super
-        into punk. You might like each other back as number one while the world doesn\'t care about
-        you two.
+        Of course, there is a chance that you and your main squeeze are a niche. Say you are both
+        into punk. You might like each other back as number one while the rest of the dating world couldn't
+        care less about you two.
     </string>
     <string name="about_background_heading">Background</string>
     <string name="about_background_body_text">
-        We\'ve all been in group situations, e.g. a class or summer camp. After a little while, the
```

Comment:
You can re-write the about part from scratch. You don't have to limit yourself to simply editing the text. I wrote that with little planning and no structure.